### DISCUSSION TITLE
Unable to fetch Products listing after adding product ,ajax unable to fetch data

### DISCUSSION THREAD

**Discussion by ashish-2790** [bagisto/bagisto#6311] [General]
Unable to fetch Products listing after adding product ,ajax unable to fetch data
![Uploading Screenshot 2022-04-19 095217.jpg…]()

**Comment by Sanjaybhattwebkul** [comment]
Hi @ashish-2790 
Can you please share more info so that I can check?
Please share the steps, which you are following.

**Comment by ashish-2790** [comment]
unable to fetch listing after adding product,customer and others, ajax
unable to fetch data PFA

On Tue, Apr 19, 2022 at 10:20 AM Sanjaybhattwebkul ***@***.***>
wrote:

> Hi @ashish-2790 <https://github.com/ashish-2790>
> Can you please share more info so that I can check
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bagisto/bagisto/discussions/6311#discussioncomment-2590047>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APLJ3PJLSDCJNEMFBMXZADTVFY3RBANCNFSM5TXSU7IQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Ashish Kumar

Senior Web Developer

Web Developement | IQwing Infotech Pvt Ltd
8219587198
***@***.***
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>

**Comment by Sanjaybhattwebkul** [comment]
@ashish-2790 
Which **version** of bagisto are you using?

**Comment by ashish-2790** [comment]
Version : v1.4.2

On Tue, Apr 19, 2022 at 10:26 AM Sanjaybhattwebkul ***@***.***>
wrote:

> @ashish-2790 <https://github.com/ashish-2790>
> Which *version* of bagisto are you using?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bagisto/bagisto/discussions/6311#discussioncomment-2590064>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APLJ3PKEXNZE7MHZYPKKSSTVFY4INANCNFSM5TXSU7IQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Ashish Kumar

Senior Web Developer

Web Developement | IQwing Infotech Pvt Ltd
8219587198
***@***.***
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>

**Comment by Sanjaybhattwebkul** [comment]
Can you share the screenshot of the error?

**Comment by ashish-2790** [comment]
Sure, here it is.

On Tue, Apr 19, 2022 at 10:32 AM Sanjaybhattwebkul ***@***.***>
wrote:

> Can you share the screenshot of the error?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bagisto/bagisto/discussions/6311#discussioncomment-2590083>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APLJ3PKV7I2GWRSO4H56HUDVFY473ANCNFSM5TXSU7IQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Ashish Kumar

Senior Web Developer

Web Developement | IQwing Infotech Pvt Ltd
8219587198
***@***.***
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>

**Comment by Sanjaybhattwebkul** [comment]
@ashish-2790 
Not, able to open the screenshot. is it possible to share a screen recording? for the same.

**Comment by ashish-2790** [comment]
Please check now.

On Tue, Apr 19, 2022 at 10:37 AM Sanjaybhattwebkul ***@***.***>
wrote:

> @ashish-2790 <https://github.com/ashish-2790>
> Not, able to open the screenshot. is it possible to share a screen
> recording? for the same.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bagisto/bagisto/discussions/6311#discussioncomment-2590098>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APLJ3PK4DBCCTAHANWEGF5LVFY5QBANCNFSM5TXSU7IQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Ashish Kumar

Senior Web Developer

Web Developement | IQwing Infotech Pvt Ltd
8219587198
***@***.***
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>

**Comment by Sanjaybhattwebkul** [comment]
@ashish-2790 
Please upload properly. still not opening, And share the screen recording if possible.

**Comment by ashish-2790** [comment]
Please check now.
 Bagisto
<https://drive.google.com/drive/folders/1vs97BaqX1Nrqb2mD8GkXHtqUozKvjcGa>


On Tue, Apr 19, 2022 at 10:47 AM Sanjaybhattwebkul ***@***.***>
wrote:

> @ashish-2790 <https://github.com/ashish-2790>
> Please upload properly. still not opening
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bagisto/bagisto/discussions/6311#discussioncomment-2590143>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APLJ3PL37P7G5DFUEKQJTF3VFY6YFANCNFSM5TXSU7IQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Ashish Kumar

Senior Web Developer

Web Developement | IQwing Infotech Pvt Ltd
8219587198
***@***.***
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>

**Comment by ashish-2790** [comment]
have you got the details


On Tue, Apr 19, 2022 at 10:53 AM ashish kumar ***@***.***>
wrote:

> Please check now.
>  Bagisto
> <https://drive.google.com/drive/folders/1vs97BaqX1Nrqb2mD8GkXHtqUozKvjcGa>
>
>
> On Tue, Apr 19, 2022 at 10:47 AM Sanjaybhattwebkul <
> ***@***.***> wrote:
>
>> @ashish-2790 <https://github.com/ashish-2790>
>> Please upload properly. still not opening
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/bagisto/bagisto/discussions/6311#discussioncomment-2590143>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/APLJ3PL37P7G5DFUEKQJTF3VFY6YFANCNFSM5TXSU7IQ>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
> Ashish Kumar
>
> Senior Web Developer
>
> Web Developement | IQwing Infotech Pvt Ltd
> 8219587198
> ***@***.***
> Create Your Own Free Signature
> <https://www.hubspot.com/email-signature-generator?utm_source=create-signature>
>


-- 
Ashish Kumar

Senior Web Developer

Web Developement | IQwing Infotech Pvt Ltd
8219587198
***@***.***
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>

**Comment by Sanjaybhattwebkul** [comment]
@ashish-2790 
Have you done, any customization?

**Comment by ashish-2790** [comment]
URL - https://ecomwebsite.iqwing.live/admin/login

Username - ***@***.***

password - ***@***.***

IP- 3.103.119.3

FTP- Username:- ***@***.***

FTP Password :- ***@***.***



On Tue, Apr 19, 2022 at 11:36 AM ashish kumar ***@***.***>
wrote:

> No not yet.
>
> On Tue, Apr 19, 2022 at 11:33 AM Sanjaybhattwebkul <
> ***@***.***> wrote:
>
>> @ashish-2790 <https://github.com/ashish-2790>
>> Have you done, any customization?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/bagisto/bagisto/discussions/6311#discussioncomment-2590292>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/APLJ3PMBT5FXDDRFMHYOTWLVFZEBNANCNFSM5TXSU7IQ>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
> Ashish Kumar
>
> Senior Web Developer
>
> Web Developement | IQwing Infotech Pvt Ltd
> 8219587198
> ***@***.***
> Create Your Own Free Signature
> <https://www.hubspot.com/email-signature-generator?utm_source=create-signature>
>


-- 
Ashish Kumar

Senior Web Developer

Web Developement | IQwing Infotech Pvt Ltd
8219587198
***@***.***
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>
